src/migrations/20240104110924-remove-unwanted-aro-topics.js

```diff
@@ -0,0 +1,30 @@
+const {
+  prepMigration,
+} = require('../lib/migration');
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, _Sequelize) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      const sessionSig = __filename;
+      await prepMigration(queryInterface, transaction, sessionSig);
+      await queryInterface.sequelize.query(
+        `DELETE FROM "ActivityReportObjectiveTopics"
+        WHERE "activityReportObjectiveId" IN (
```

Comment:
Setting aside logic, I think we'd want to write it like this:
```
DELETE FROM  "ActivityReportObjectiveTopics" arot
USING "ActivityReportObjectives" aro
JOIN "ZALActivityReportObjectiveTopics" zaro
  ON dml_txid = '00000000-0000-0000-0000-000002266502'
  AND aro.id = (zaro.new_row_data->>'activityReportObjectiveId')::int
JOIN "ActivityReports" ar
  ON aro."activityReportId" = ar.id
  AND ar.version = 1
WHERE arot."activityReportObjectiveId" = aro.id
  AND data_id = arot.id
```
There are slight differences in the results because 10 of the ActivityReportObjectiveTopics (AROTs) were actually added later to AROs that were created during the migration. If we want to just delete _all_ AROTs associated with the AROs created in the initial migration, even if the topics were associated later, then we'd need a CTE, but I'm not sure if we do want to do that.

Reply:
Per our discussion, we truly don't want _any_ AROTs for those AROs, so the correct logic is:
```
DELETE FROM  "ActivityReportObjectiveTopics" arot
USING "ActivityReportObjectives" aro
JOIN "ZALActivityReportObjectiveTopics" zaro
  ON dml_txid = '00000000-0000-0000-0000-000002266502'
  AND aro.id = (zaro.new_row_data->>'activityReportObjectiveId')::int
JOIN "ActivityReports" ar
  ON aro."activityReportId" = ar.id
  AND ar.version = 1
WHERE arot."activityReportObjectiveId" = aro.id
```